File: PL036.java
import java.util.Scanner;

public class PL036 {

    public static void main(String[] args) {
        String s;
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the string to convert into the title case ");
        s = scanner.nextLine();
        scanner.close();
        System.out.println("The title case string is : ");
        System.out.println(transform(s));
    }

    public static String transform(String s) {
        char[] temp = s.toCharArray();
        boolean flag = true;
        String result = "";

        for (int i = 0; i < s.length(); i++) {
            if (temp[i] == ' ') {
                flag = true;
                result += temp[i];
                continue;
            }

            if (flag) {
                if (temp[i] >= 'a' && temp[i] <= 'z')
                    temp[i] -= 32;
                flag = false;
            }

            result += temp[i];
        }

        return result;
    }
}

File: PL036.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

string transform(string s)
{
    string result = s;
    bool flag = true;

    for (int i = 0; i < result.length(); i++)
    {
        if (result[i] == ' ')
        {
            flag = true;
            continue;
        }

        if (flag)
        {
            if (result[i] >= 'a' && result[i] <= 'z')
                result[i] -= 32;
            flag = false;
        }
    }

    return result;
}

int main()
{

    string s;
    cout << "Enter the string to convert it into the title case : " << endl;
    // cin >> s;
    getline(cin, s);
    cout << "The title case string is : " << transform(s) << endl;

    return 0;
}